__tests__/app.test.js

```diff
@@ -106,3 +106,47 @@ describe("GET /api/articles", () => {
       });
   });
 });
+describe("POST /api/articles/:article_id/comments", () => {
+  test("should return 201 and respond with added comment", () => {
+    return request(app)
+      .post("/api/articles/1/comments")
+      .send({ username: "rogersop", body: "I love this article very much!!" })
+      .expect(201)
+      .then(({ body }) => {
+        expect(body.comment).toHaveLength(1);
+        expect(body.comment[0]).toHaveProperty(
+          "comment_id",
+          expect.any(Number)
+        );
+        expect(body.comment[0]).toHaveProperty("body", expect.any(String));
+        expect(body.comment[0]).toHaveProperty("votes", expect.any(Number));
+        expect(body.comment[0]).toHaveProperty("author", expect.any(String));
+        expect(body.comment[0]).toHaveProperty(
```

Comment:
Please send this in an object not an array. Also you can test for almost all of the exact data since you know what you are posting